pvanalytics/tests/test_features.py

```diff
@@ -8,7 +8,11 @@
 
 @pytest.fixture
 def quadratic():
-    """Downward facing quadratic with vertex at (0, 1000) and roots at +/-1."""
+    """Downward facing quadratic.
+
+    Vertex at (30.5, 1000) and roots at 0, 59.
```

Comment:
`0, 59` is the index where the roots are located?  +/-1 are the x-values.

Reply:
I'm thinking about it as the index of the Series being the x-values. Having the documentation in those terms makes it easier (for me) to write additional tests that use the fixture.

Reply:
Makes sense. Maybe "Downward facing quadratic with vertex at index position 30" - and redefine `q` as in next comment

Reply:
```suggestion
    Vertex at index 30 and roots at indices 0, 60.
```